autopush/endpoint.py

```diff
@@ -12,6 +13,13 @@
 from twisted.python import log
 
 
+def MakeEndPoint(fernet, endpoint_url, uaid, chid):
```

Comment:
What do you think about making this a method on `AutopushSettings`, since all callers use `ap_settings.fernet` and `ap_settings.endpoint_url`?